docs/ibmse.md

```diff
@@ -86,10 +113,22 @@ metadata:
 spec:
   # omitted all the rest of config
   # ...
-
+  kbsAttestationPolicyConfigMapName: ibmse-attestation-policy
+  kbsResourcePolicyConfigMapName: ibmse-resource-policy
+  kbsServiceType: NodePort
   # IBMSE settings
   ibmSEConfigSpec:
     certStorePvc: ibmse-pvc
 ```
-
 The `certStorePvc` has to match the aforementioned PVC name.
+**Note:** 
+- ibmse tee need the kbs service run in NodePort type, if the https is enabled, please make sure include the worker node ips to the `[alt_names]` section, here is the document about how to [generate a self signed certificate](https://github.com/confidential-containers/trustee/blob/main/kbs/docs/self-signed-https.md#generate-a-self-signed-certificate)
```

Comment:
Why do you need a NodePort service type? Is the coco pod running inside the cluster?

Reply:
yes, the coco pod running inside the cluster.
In coco pod we use the attestation agent and CDH service to communicate with KBS. The AA and CDH service are run as the systemd service, use the short kbs service, AA and CDH service will report DNS name not found issue.
So in my test env, I use node port service type, so that the AA and CDH service can access the kbs service from the peerpod VM.

Reply:
peerpod VM is the HOST, where the pod container workload is running.